src/Symfony/Component/Finder/Comparator/DateComparator.php

```diff
@@ -9,6 +9,8 @@
  * file that was distributed with this source code.
  */
 
+declare(strict_types=1);
```

Comment:
I guess this change should be reverted. There's an ongoing discussion about adding it or not to new code (see #28423).

Reply:
waiting for approval of RFC ❤️